Gaming-Setup-Shop/src/main/java/com/GamingSetupShop/models/entities/Customer.java

```diff
@@ -0,0 +1,40 @@
+package com.GamingSetupShop.models.entities;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.Set;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Data
+@Entity
+@Table(name = "customers")
+public class Customer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "first_name")
+    private String firstName;
```

Comment:
If your CustomerDTO has this property as customerFirstName, this means we don't pay attention to consistency. Please be aware of that.